### ISSUE TITLE
Illegal Instruction Fault on executing D extension instructions on a rv32 machine.

### ISSUE THREAD

**Issue by pawks** [rems-project/sail-riscv#96]
On executing an instruction from the D extension in the `rv32` model an illegal instruction fault is raised.  The `misa` reset value for a `rv32` machine is `0x40141105`. This indicates that the D extension is disabled whereas the reset value should be `0x4014110D`. The model encounters an illegal instruction fault even after setting the `misa` to the correct value.

**Comment by jrtc27** [comment]
Yes, this is known, D is only supported on RV64 in the current model https://github.com/riscv/sail-riscv/blob/4768f62f9a779b254bc7dcd4a680ade794db92f4/Makefile#L10
